wake_vad_node:
  ros__parameters:
    access_key: ""
    keyword_path: "/home/ubuntu/rover_ws/src/wake_vad_cpp/wake_sound"
    porcupine_model_path: "/home/ubuntu/rover_ws/src/wake_vad_cpp/resource/porcupine_params_ko.pv"
    audio_device_index: -1
    sensitivity: 0.5
    vad_threshold: 0.5
    silence_duration: 1.5
    max_record_duration: 10.0
    wav_output_dir: "/home/ubuntu/rover_ws/src/wake_vad_cpp/record_sound"
    listen_timeout: 8.0
    vad_model_path: "/home/ubuntu/rover_ws/src/wake_vad_cpp/resource/silero_vad.onnx"
    wake_prompt_tts_enabled: true
    wake_prompt_tts_topic: "/llm/response"
    wake_prompt_tts_text: "네, 말씀하세요."
    wake_prompt_wait_tts_done_enabled: true
    wake_prompt_wait_timeout_sec: 15.0
    tts_playback_done_topic: "/tts/playback_done"
    save_wav_for_debug: false
